dictionaries/bash/src/bash-words.txt

```diff
@@ -1,9 +1,13 @@
+--noprofile
```

Comment:
`--noprofile` will match `--noprofile`, but not `noprofile`.

If you think it might be used without the `--`, then use `noprofile`.

Reply:
I don't really know. I'm fine with both. How about leaving it like this until somebody complains?

Reply:
I'm fine with that.